.github/workflows/multibuild.yaml

```diff
@@ -50,29 +50,35 @@ jobs:
     defaults:
       run:
         working-directory: ./packages/dart/sshnoports
+        shell: bash
     strategy:
       matrix:
         include:
           - os: ubuntu-latest
             output-name: sshnp-linux-x64
             ext: ""
             bundle: "shell"
+            platform: "linux"
           - os: ubuntu-24.04-arm
             output-name: sshnp-linux-arm64
             ext: ""
             bundle: "shell"
+            platform: "linux"
           - os: macos-13
             output-name: sshnp-macos-x64
             ext: ""
             bundle: "shell"
+            platform: "macos"
           - os: macos-14
             output-name: sshnp-macos-arm64
             ext: ""
             bundle: "shell"
-          - os: windows-latest
+            platform: "macos"
+          - os: windows-signer
```

Comment:
Using the self-hosted runner which has certs available for signing our binaries.